### PR TITLE
Update importing.md

### PR DESCRIPTION
**Pull request by bpaulien**
title case, and add missing words.

### CODE REVIEW

File: docs/transactions/importing.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 There are a couple ways to get transactions into the system. For now, automatic transaction downloading is not available but it will be available very soon.
 
-## Import financial files
+## Import Financial Files
 
-A quick way to import transactions is to login to your bank's website and download a file. Actual supports importing QIF, OFX, and QFX files. Your bank probably allows you to download one of these formats (OFX/QFX is recommended).
+A quick way to import transactions is to login to your bank's website and download a file. Actual supports importing QIF, OFX, and QFX files. Your bank probably allows you to download one of these formats (OFX/QFX is recommended.)
```
Comment (RubenOlsen) [inline docs/transactions/importing.md:7]: Actually - the full stop needs to be outside the parenthesis.

**Review by RubenOlsen — CHANGES_REQUESTED** [review]
Everything is good except a small nit-pick.

File: docs/transactions/importing.md (hunk 1 above)
Comment (bpaulien) [reply docs/transactions/importing.md]: Ok, I made the change.

**Review by bpaulien — COMMENTED** [review]
fixed punctuation

**Review by RubenOlsen — APPROVED** [review]

**Pull request merged**